### ISSUE TITLE
Add `config.with_reporters` function?

### ISSUE THREAD

**Issue by mooreryan** [maxdeviant/startest#4]
Would you be interested in a PR to add a `config.with_reporters` function? E.g., 

```gleam
/// Updates the given `Config` with the specified `reporters`.
pub fn with_reporters(config: Config, reporters: List(Reporter)) -> Config {
  Config(..config, reporters: reporters)
}
```

This would give each of the `Config` record fields a corresponding `with_*` function.

**Comment by maxdeviant** [comment]
Huh, I don't remember why I didn't add one when I added the config functions.

But since the other fields have one, I think `config.with_reporters` would make sense to add.

**Comment by mooreryan** [comment]
(Addressed by #5)
